packages/ember-flight-icons/babel.config.json

```diff
@@ -1,9 +1,28 @@
 {
-  "presets": [["@babel/preset-typescript"]],
   "plugins": [
+    [
+      "@babel/plugin-transform-typescript",
+      {
+        "allExtensions": true,
+        "onlyRemoveTypeImports": true,
+        "allowDeclareFields": true
+      }
+    ],
     "@embroider/addon-dev/template-colocation-plugin",
-    ["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],
-    ["@babel/plugin-proposal-decorators", { "legacy": true }],
-    "@babel/plugin-transform-class-properties"
```

Comment:
unfortunately, we can't remove these – they are still needed for older codebases (such as `atlas`) to work. I ended up re-adding them in `components`.